projects/bank-frontend/components/podinfo/podinfo.cue

```diff
@@ -0,0 +1,27 @@
+package holos
+
+holos: Podinfo.BuildPlan
+
+// Reference the correct environment namespace for frontend applications
+_namespace: BankOfHolos.configuration.environments[EnvironmentName].frontend.namespace
+
+Podinfo: #Helm & {
+	Name: "podinfo"
+    Namespace: _namespace
+    // Ensure metadata.namespace is added to all resources with kustomize.
+	KustomizeConfig: Kustomization: namespace: _namespace
+	Chart: {
+		version: "6.6.2"
+		repository: {
+			name: "podinfo"
+			url:  "https://stefanprodan.github.io/podinfo"
+		}
+	}
+
+    // List comprehension acting as a Switch statement: https://cuetorials.com/patterns/switch/
+    // Use 3 replicas in prod, and 1 in all other environments
+    Values: replicaCount: [
+        if EnvironmentName =~ "^prod" {3},
+        1,
+    ][0] // Select the first element in the list
```

Comment:
Let's not use a list comprehension like this, while it works and I eventually figured it out, it's not what I had in mind.

To vary by prod or nonprod, I was thinking we look things up in a struct, like this:

```cue
#TierValues: [TIER=string]: replicaCount: int & >=0

PodInfo: TierValues: #TierValues
PodInfo: TierValues: nonprod: replicaCount: 1
PodInfo: TierValues: prod: replicaCount: 3
```

Then in `projects/bank-frontend/components/podinfo/podinfo.cue` we can dig into that struct:

```cue
let TIER = BankOfHolos.Environments[EnvironmentName].tier
Podinfo: Values: Podinfo.TierValues[TIER]
```